packages/kit/src/components/Hardware/Hardware.tsx

```
@@ -1,4 +1,4 @@
-import { useCallback, useEffect, useMemo, useState } from 'react';
+import { ReactNode, useCallback, useEffect, useMemo, useState } from 'react';
```

ESLint

[error] Import "ReactNode" is only used as types.

(@typescript-eslint/consistent-type-imports)